lib/components/search/Autocomplete.js

```diff
@@ -10,44 +11,41 @@ import SingleValue from './SingleValue';
 import ValueContainer from './ValueContainer';
 import Menu from './MenuFilter';
 import MultiValue from './MultiValue';
+import OptionFilter from './OptionFilter';
 
 const styles = theme => ({
   root: {
     position: 'relative',
     minWidth: '70px',
   },
+  wide: {
+    minWidth: '300px',
+  },
 });
 
 const IndicatorSeparator = () => null;
 
 const ClearIndicator = () => null;
 
-function Option(props) {
+const OptionContainer = props => {
+  const { children, innerRef, innerProps, isFocused, isSelected } = props;
   return (
     <MenuItem
-      buttonRef={props.innerRef}
-      selected={props.isFocused}
+      buttonRef={innerRef}
+      selected={isFocused}
       component="div"
       style={{
-        fontSize: '0.8rem',
-        fontWeight: props.isSelected ? 500 : 400,
-        padding: '0 5px',
-        borderLeft: `4px solid ${
-          props.data.selected
-            ? '#0091eb'
-            : props.data.chained
-              ? '#ff6350'
-              : '#9e9e9e'
-        }`,
-        marginLeft: '2px',
-        marginBottom: '2px',
+        fontWeight: isSelected ? 500 : 400,
```

Comment:
Since this component is already being wrapped by the `withStyles` HOC, you could move these styles to the definition of styles up in line 16 https://github.com/opentargets/ot-ui/pull/50/files#diff-147a2dc3da222dc04f2d51d7e66a1dc9R16

Reply:
Nevermind, these styles depend on a prop.